ThunderTable/TableViewController.swift

```diff
@@ -570,7 +577,53 @@ open class TableViewController: UITableViewController, UIContentSizeCategoryAdju
             set(indexPath: indexPath, selected: false)
         }
     }
-	
+    
+    //MARK: -
+    //MARK: Scroll Offset Management
+    //MARK:
+    
+    private lazy var scrollOffsetManager: ScrollOffsetManager = {
```

Comment:
Up to you here, there's nothing wrong with doing it with `= { }()` in case you add properties in the future. But atm could simply do `private let scrollOffsetManager = ScrollOffsetManager()`

Reply:
Yeah let's go with that. It's memory footprint should be tiny anyways!